frontend/src/main/components/Ride/RideForm.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import { Button, Form } from 'react-bootstrap';
+import { useForm } from 'react-hook-form'
+import { useNavigate } from 'react-router-dom';
+function RideForm({ initialContents, submitAction, buttonLabel = "Create" }) {
+
+    
+    // Stryker disable all
+    const {
+        register,
+        formState: { errors },
+        handleSubmit,
+    } = useForm(
+        { defaultValues: initialContents || {}, }
+    );
+    // Stryker enable all
+    const navigate = useNavigate();
+
+    //const testIdPrefix = "RideForm";
+
+    return (
+
+        <Form onSubmit={handleSubmit(submitAction)}>
+
+            {initialContents && (
+                <Form.Group className="mb-3" >
+                    <Form.Label htmlFor="id">Id</Form.Label>
+                    <Form.Control
+                        data-testid={"RideForm-id"}
```

Comment:
```suggestion
                        data-testid={`${testid}-id`}
```
Once you uncomment the line above